### PR TITLE
Upgrade influxql

### PR DESCRIPTION
**Pull request by docmerlin**

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -1114,7 +1114,7 @@
     "github.com/influxdata/influxdb",
     "github.com/influxdata/influxdb/client",
     "github.com/influxdata/influxdb/client/v2",
-    "github.com/influxdata/influxdb/influxql",
```
Comment (lesam) [inline Gopkg.lock:1117]: Why do we still have a Gopkg if we also have a `go.mod` ? Can we delete this file?

File: go.mod [hunk 2]
```diff
@@ -233,8 +240,6 @@ require (
 
 replace gopkg.in/fsnotify.v1 => github.com/fsnotify/fsnotify v1.4.2
 
-replace github.com/influxdata/influxdb => github.com/influxdata/influxdb v1.1.4
```
Comment (lesam) [inline go.mod:236]: 👍  Wow

File: server/server.go [hunk 3]
```diff
@@ -12,11 +12,14 @@ import (
 	"sync"
 	"time"
 
-	"github.com/influxdata/influxdb/influxql"
+	"go.uber.org/zap"
+
+	"github.com/influxdata/influxdb/query"
```
Comment (lesam) [inline server/server.go:17]: Does kapa use `gci` to enforce import grouping yet?

File: task_master.go [hunk 4]
```diff
@@ -267,6 +269,10 @@ type TaskMaster struct {
 	wg      sync.WaitGroup
 }
 
+func (tm *TaskMaster) WritePointsPrivileged(ctx tsdb.WriteContext, database, retentionPolicy string, consistencyLevel imodels.ConsistencyLevel, points []imodels.Point) error {
```
Comment (lesam) [inline task_master.go:272]: Are we adding this to implement an interface? Can we adjust whatever requires that interface to not require unused methods?

**Review by lesam — APPROVED** [review]
Wow, it's shocking how little changed here - score one for API stability

File: task_master.go (hunk 4 above)
Comment (docmerlin) [reply task_master.go:272]: It is an interface in influxdb 1.9.6 where it is actually used.

File: Gopkg.lock (hunk 1 above)
Comment (docmerlin) [reply Gopkg.lock:1117]: I think we can.   I'll make a separate PR to do that.

File: task_master.go (hunk 4 above)
Comment (lesam) [reply task_master.go:272]: Does kapa allow `SELECT INTO` ? Do you get a good error message if you try to use that?

Comment (docmerlin) [reply task_master.go:272]: It does allow it but it just the same as having run it on the target influx machine

**Review by lesam — APPROVED** [review]

**Pull request merged**